behaviors/TranslatableModel.php

```diff
@@ -21,6 +21,25 @@
  */
 class TranslatableModel extends TranslatableBehavior
 {
+    public function __construct($model)
+    {
+        parent::__construct($model);
+
+        $model->morphMany['translations'] = [
+            'RainLab\Translate\Models\Attribute',
+            'name' => 'model'
+        ];
+    }
+
+    public function scopeTransWith($query)
```

Comment:
What's the point of this scope? Why wouldn't developers just use `with(['translations'])` by itself?

Reply:
Hi @LukeTowers, you are right. scopeTransWith could be chained to other scopes, but "with" option is much more flexible. I'll remove scopeTransWith without any regrets.

Reply:
@LukeTowers scopeTransWith method removed